script.js

```diff
@@ -203,8 +203,18 @@ let projectData = [
     projectName: "Text to Voice Generator",
     projectImage: "assets/GIFs/textToVoice",
     projectUrl: "public/textToVoice.html",
+  },
+  {
+    projectName: "Progress bar",
+    projectImage: "assets/GIFs/progressbar.png",
+    projectUrl: "public/progressbar.html",
   }
-];
+  ,
+  {
+    projectName: "Herry Potter Characters",
+    projectImage: "assets/Images/herry.png",
+    projectUrl: "public/herry.html",
```

Comment:
Not working. Fix the name.

Reply:
can I fork again??

Reply:
please look into #1020  my laptop get crashed that's why I did .